org.endlessos.Key.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: org.endlessos.Key
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: /app/bin/kolibri-gnome
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+  - --system-talk-name=org.endlessos.Key.Daemon
```

Comment:
system? How do you make sure it exists?

Reply:
Good question!

Endless OS provides features and services from [eos-kolibri](https://github.com/endlessm/eos-kolibri) to manage the content with system resource.  Endless Key checks environment variable [`ENDLESS_KEY_USE_SYSTEM_INSTANCE`](https://github.com/endlessm/endless-key-flatpak/blob/v0.1/src/libkolibri_daemon_dbus/kolibri-daemon-dbus-utils.c#L112) before talks to the named service on the system bus.

Reply:
Given that Flathub !- Endless, how would that work on, say, Fedora.

Reply:
aiui, on non-Endless systems, a session daemon is used instead of the system daemon. This performs the same functions, but doesn’t allow sharing of downloaded content between users.